HealthcareSystem/Backend/Repository/SurveyRepository/MySqlSurveyRepository/MySqlSurveyRepository.cs

```diff
@@ -18,24 +18,147 @@ public MySqlSurveyRepository(MyDbContext context)
 
         public void AddSurvey(Survey survey)
         {
-            _context.Surveys.Add(survey);
+            _context.SurveysAboutDoctor.Add(survey.SurveyAboutDoctor);
+            _context.SurveysAboutMedicalStaff.Add(survey.SurveyAboutMedicalStaff);
+            _context.SurveysAboutHospital.Add(survey.SurveyAboutHospital);
             _context.SaveChanges();
         }
 
-        public Survey GetSurveyById(int id)
+        public List<SurveyResult> GetSurveyResultsAboutDoctor(string jmbg)
```

Comment:
A method contains too many lines of code. Generally, any method longer than ten lines should make you start asking questions.  

If you feel the need to comment on something inside a method, you should take this code and put it in a new method. Even a single line can and should be split off into a separate method, if it requires explanations. And if the method has a descriptive name, nobody will need to look at the code to see what it does.

To reduce the length of a method body, use Extract Method ( https://refactoring.guru/extract-method ).